src/Symfony/Component/WebLink/composer.json

```diff
@@ -20,7 +20,7 @@
     },
     "require": {
         "php": ">=8.0.2",
-        "psr/link": "^1.0"
+        "psr/link": "^1.1"
```

Comment:
Shall we add a conflict rule for `ext-psr`? If we bump here, the component will break if (any released version of) `ext-psr` is installed.

Reply:
I wouldn't care, that's not our problem